File: js/Car.js
class Car {
  id;
  license;
  driver;
  passenger;

  constructor(license, driver, passenger) {
    this.id = new Date().getDate();
    this.license = license;
    this.driver = driver;
    this.passenger = passenger;
  }

  printDataCar() {
    console.log(`License: ${this.license}, driver: ${this.driver.name}, passengers: ${this.passenger}`)
  }
}

File: python/car.py
import random;

class Car: 
  id = int
  license = str
  driver = str
  passengers = int
  
  def __init__(self,license,driver, passengers):
    self.id = random.randint(0,50000)
    self .license = license
    self.driver = driver
    self.passengers = passengers
  
  def get_info(self):
    print('license:' + self.license + ', driver:' + self.driver)
  